evap/staff/views.py

```diff
@@ -2164,12 +2178,72 @@ def user_import(request):
         "staff_user_import.html",
         {
             "excel_form": excel_form,
+            "decision_form": decision_form,
             "importer_log": importer_log,
             "test_passed": test_passed,
         },
     )
 
 
+def export_users(file, users):
```

Comment:
Maybe the same component that initially transforms the `xlsx` file to our internal representation should also be responsible for building a `xlsx` file again, so we just have the logic encapsulated in one place.